Formula/ghc.rb

```diff
@@ -4,6 +4,7 @@ class Ghc < Formula
   url "https://downloads.haskell.org/~ghc/8.10.3/ghc-8.10.3-src.tar.xz"
   sha256 "ccdc8319549028a708d7163e2967382677b1a5a379ff94d948195b5cf46eb931"
   license "BSD-3-Clause"
+  revision 2
```

Comment:
```suggestion
  revision 1
```
The formula currently has no revision, so this should only go to `1` now.

Reply:
Ok, makes sense.